CMakeLists.txt

```diff
@@ -314,6 +314,10 @@ if(USE_MKLDNN)
   set(INSTALL_MKLDNN ON)
 endif()
 
+if(USE_CPP_PACKAGE)
```

Comment:
This change seems unrelated to this PR?

Reply:
Yes, you're right - accidentally brought in external commits due to an incorrect rebase. Will fix ASAP.

Reply:
Fixed.